keyboards/gurindam/readme.md

```diff
@@ -0,0 +1,15 @@
+# gurindam
+
+![Gurindam](https://cdn.discordapp.com/attachments/406032987243806721/922514068265369611/IMG_20211220_222843.jpg)
+
+A keyboard inspired by Arisu which in turn by Lyn's EM7 and TGR Alice.
+
+* Keyboard Maintainer: [Ibnu D. Aji](https://github.com/ibnuda)
+* Hardware Supported: Promicro
+* Hardware Availability: [PCB & Stacked Acrylic Case](https://github.com/ibnuda/gurindam) & private group-buy.
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```